core/src/main/java/tc/oc/pgm/variables/types/PlayerLocationVariable.java

```diff
@@ -30,6 +44,13 @@ protected double getValueImpl(MatchPlayer player) {
       case VEL_X -> player.getBukkit().getVelocity().getX();
       case VEL_Y -> player.getBukkit().getVelocity().getY();
       case VEL_Z -> player.getBukkit().getVelocity().getZ();
+      case TARGET_X -> intersection(player).getBlock().getX();
```

Comment:
These all still have potential for NPE if a map author forgets to check for HAS_TARGET = 1

Reply:
right